StrongReferenceDemo/MovieListViewController.swift

```diff
@@ -66,7 +68,7 @@ final class MovieListViewController: UITableViewController {
         }
 
         cell.node = node
-        cell.textLabel?.text = cell.statusText()
```

Comment:
See below: Let's remove the `statusText` property and just make this `cell.textLabel?.text = `node.name`